.github/workflows/main.yml

```
@@ -125,7 +124,7 @@ jobs:
       fail-fast: false
       matrix:
         # mypy 1.5 dropped support for Python 3.7
-        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
+        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12']
```

```suggestion
        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11", "3.12"]
```
Just keeping quote style consistent